File: browserwn/src/main/java/org/sqlunet/browser/wn/MainActivity.java
/*
 * Copyright (c) 2023. Bernard Bou <1313ou@gmail.com>
 */

package org.sqlunet.browser.wn;

import android.os.Bundle;

import androidx.annotation.Nullable;

public class MainActivity extends org.sqlunet.browser.MainActivity
{
	@Override
	protected void onPostCreate(@Nullable final Bundle savedInstanceState)
	{
		super.onPostCreate(savedInstanceState);
		Oewn.hook(this);
	}
}

File: browserwn/src/main/java/org/sqlunet/browser/wn/MainActivity.kt
/*
 * Copyright (c) 2023. Bernard Bou <1313ou@gmail.com>
 */
package org.sqlunet.browser.wn

import org.sqlunet.browser.MainActivity
import org.sqlunet.browser.wn.Oewn.hook

class MainActivity : MainActivity() {

    override fun onStart() {
        super.onStart()
        hook(this)
    }
}
